README.md

```diff
@@ -5095,6 +5116,7 @@ Callback for the periodic and non periodic vehicle data read function.
 |`emergencyEvent`|EmergencyEvent|False|Information related to an emergency event (and if it occurred)|
 |`clusterModeStatus`|ClusterModeStatus|False|The status modes of the cluster|
 |`myKey`|MyKey|False|Information related to the MyKey feature|
+|`windowStatus`|Boolean|False|Information related to window status|
```

Comment:
change to Window status struct

Reply:
@IGapchuk fixed in https://github.com/smartdevicelink/rpc_spec/pull/250/commits/65b609a32c61bd643297a95a75f1faae7735811b